src/components/comment/_comment_event.scss

```
@@ -0,0 +1,76 @@
+@include euiPanel($selector: '.euiCommentEvent--regular');
+
+.euiCommentEvent {
+  font-size: $euiFontSizeS;
+  overflow: hidden;
+
+  .euiCommentEvent__event {
+    padding-right: $euiSizeXS;
+  }
+
+  .euiCommentEvent__header {
+    @include euiBreakpoint('xs', 's') {
+      margin: -($euiSizeXS);
+
+      >div {
+        margin: $euiSizeXS;
+      }
+
+      .euiCommentEvent__headerTimestamp {
+        padding-top: $euiSizeXS * .5;
+      }
+    }
+
+    align-items: center;
+
+    .euiCommentEvent__headerData>div {
```

Watch for spacing here 

```suggestion
    .euiCommentEvent__headerData > div {
```